project/LinkerdBuild.scala

```diff
@@ -69,7 +69,7 @@ object LinkerdBuild extends Base {
   }
 
   val configCore = projectDir("config")
-    .withTwitterLibs(Deps.finagle("core"), Deps.finagle("thrift"))
+    .withTwitterLibs(Deps.finagle("core"))
```

Comment:
`ThriftProtocolDeserializer` got moved to `linkerd-protocol-thrift`  as part of this PR.